internal/exec.go

```diff
@@ -94,14 +94,19 @@ func (e *ExecCommand) getCluster() {
 		return
 	}
 
-	if len(list.ClusterArns) > 0 {
-		var clusterNames []string
-		for _, c := range list.ClusterArns {
-			arnSplit := strings.Split(*c, "/")
-			name := arnSplit[len(arnSplit)-1]
-			clusterNames = append(clusterNames, name)
-		}
+	var clusterNames []string
+	for _, c := range list.ClusterArns {
+		arnSplit := strings.Split(*c, "/")
+		name := arnSplit[len(arnSplit)-1]
+		clusterNames = append(clusterNames, name)
+	}
 
+	switch {
+	case len(clusterNames) == 1:
```

Comment:
Could we add a message here that tells the user it's auto-selecting the single cluster, think it might then appear less "magic" to the end-user? Something like this perhaps
```
fmt.Println(yellow(fmt.Sprintf("Single cluster found in region (%s), selecting by default...", clusterNames[0])))
```

Reply:
Makes sense. Will update the code